orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/functions/ArtifactExpressionFunctionProvider.java

```diff
@@ -42,16 +45,21 @@ public Functions getFunctions() {
     return new Functions(
         new FunctionDefinition(
             "triggerResolvedArtifact",
-            "Looks up the an artifact in current execution given its name. If multiple artifacts are found, only 1 will be returned.",
+            "Looks up a resolved artifact in current execution given its name. If multiple artifacts are found, only 1 will be returned.",
```

Comment:
To match the change below?
```suggestion
            "Looks up a resolved artifact in the current execution trigger given its name. If multiple artifacts are found, only 1 will be returned.",
```